pkg/spoke/apply/create_only_apply.go

```diff
@@ -27,7 +26,7 @@ func (c *CreateOnlyApply) Apply(ctx context.Context,
 	gvr schema.GroupVersionResource,
 	required *unstructured.Unstructured,
 	owner metav1.OwnerReference,
-	applyOption *workapiv1.ManifestConfigOption,
```

Comment:
just thinking whether we should required.SetOwnerReferences([]metav1.OwnerReference{owner}) out of this func, so we do not need to input owner